client/my-sites/customer-home/mastering-gutenberg-card/index.jsx

```diff
@@ -0,0 +1,160 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { localize } from 'i18n-calypso';
+import { Card, Button, Dialog } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import ButtonGroup from 'components/button-group';
+import CardHeading from 'components/card-heading';
+import Gridicon from 'components/gridicon';
+import { localizeUrl } from 'lib/i18n-utils';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class MasteringGutenbergCard extends Component {
+	state = {
+		showDialog: false,
+	};
+
+	handleDialogClosure = () => {
+		this.setState( {
+			showDialog: false,
+			adjustingBlocks: false,
+			navigatingBlocks: false,
+			addingLinks: false,
+			usingImages: false,
+		} );
+	};
+
+	handleAdjustingBlocksClick = () => {
```

Comment:
I'm not sure, but we may be able to simplify these methods to just one, if we make state something like

```js
{ showDialog: true,  activeDialog: 'adjustingBlocks' }
```

This would also mean the `if else if` blocks at L60 could be a simpler [switch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) statement checking on `activeDialog`.

Reply:
That works well, thanks! :)

Reply:
That works well, thanks! :)